lib/active_fedora/indexing/descendant_fetcher.rb

```diff
@@ -90,10 +96,9 @@ def prioritized_object?
         end
 
         def add_self_to_partitioned_uris
-          if prioritized_object?
-            partitioned_uris[:priority] << rdf_resource.subject
-          else
-            partitioned_uris[:other] << rdf_resource.subject
+          rdf_graph_models.each do |model|
```

Comment:
If we're not using the `prioritized_object?` method anymore, should it be deleted?